test/e2e/context/tmc/context_tmc_suite_test.go

```diff
@@ -29,7 +29,7 @@ var (
 
 const prefix = "ctx-tmc-"
 const prefixK8s = "ctx-k8s-"
-const maxCtx = 25
+const maxCtx = 5 // setting few number of context's as TMC context creation is slow and takes time because its dependent on TMC API's
```

Comment:
Was this for testing, or do you want to keep it at 5?

Reply:
Its not for testing, we need to keep it, i had added stress tests, but they are taking really long time as we are depending on TMC endpoint.